CHANGELOG.md

```diff
@@ -79,6 +79,7 @@
 
 ### Chore & Maintenance
 
+- `[docs]` Fix message property in custom matcher example to return a function instead of a constant.
```

Comment:
Missing link to this pr

Reply:
🙈

Reply:
Fixed https://github.com/facebook/jest/pull/7429